models/agent.go

```diff
@@ -48,6 +48,14 @@ type Agent struct {
 	ID           int32     `reform:"id,pk"`
 	Type         AgentType `reform:"type"`
 	RunsOnNodeID int32     `reform:"runs_on_node_id"`
+
+	// TODO Does it really belong there? Remove when we have agent without one.
+	ListenPort *uint16 `reform:"listen_port"`
+}
+
+// NameForSupervisor is a name of agent for supervisor.
+func (a *Agent) NameForSupervisor() string {
```

Comment:
maybe it's better to use a function instead of a method?
```suggestion
func NameForSupervisor(type, port string) string {
```

Reply:
Good point!